tensorboard/webapp/core/effects/core_effects_test.ts

```diff
@@ -208,7 +214,20 @@ describe('core_effects', () => {
 
         action.next(onAction);
         httpMock.expectOne('data/plugins_listing').flush(pluginsListing);
-        expect(dispatchSpy).toHaveBeenCalledTimes(3);
+        fetchRunsSubjects[0].next([{id: '1', name: 'Run 1'}]);
```

Comment:
Up to you but I don't think it's necessary to check all the actions here.  All you really care about in this test is "did something trigger?" rather than "How did it trigger?".

Reply:
There are data manipulation (convert backend data is to the frontend data structure) so I think the "how" part is a bit important here.